core/src/main/scala/org/ensime/core/Project.scala

```diff
@@ -62,7 +65,7 @@ class Project(
     // make sure the "reliable" dependencies are cleaned up
     Try(classfileWatcher.shutdown())
     Try(sourceWatcher.shutdown())
-    Try(searchService.shutdown())
+    Try(Await.ready(searchService.shutdown(), DB_SHUTDOWN_TIMEOUT))
```

Comment:
I'm tempted to just not wait on this. The reason for all the `Try`s is that we don't care about the outcome, we just want to take best endeavours to shut down cleanly before it annoys the user and they force the kill.